source/shared/net/desktop-wallet/create-credentials-file.rst

```diff
@@ -0,0 +1,51 @@
+
+.. _create-credentials-file:
+
+=========================
+Create a credentials file
+=========================
+
+.. contents::
+   :local:
+   :backlinks: none
+   :depth: 1
+
+
+This topic describes how you create and export a file with credentials. For information about adding more credentials to an account, see :ref:`Add credentials to an account <multi-credentials>`.
+
+Create and export a file with credentials
+=========================================
+
+#. First, you need the address of the account your credentials will be added to. If you have the account in you address book, you can copy it from there. Otherwise, you'll have to ask the address creator to send you a copy of the account address.
+
+#. Go to **Multi Signature Transactions** in the Desktop Wallet.
+
+#. Select **Export a key**, and then select **Account credentials**.
+
+#. Select the identity you want to add to the account. The identity must have been created on the Ledger that's connected to the computer. Otherwise, you'll see an error message. Select **Continue**.
+
+#. Past the address of the account that you want to add credentials to, and then give the account a name. This is the name that will be shown in your list of accounts. Select **Continue**.
```

Comment:
```suggestion
#. Paste the address of the account that you want to add credentials to, and then give the account a name. This is the name that will be shown in your list of accounts. Select **Continue**.
```

Reply:
I guess technically there is the option to reveal attributes at this stage as well. Do we want to write something about that?